shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java

```diff
@@ -141,6 +147,16 @@ public void setShimmerAnimationDuration(int durationMillis) {
         resetIfStarted();
     }
 
```

Comment:
Would you also create a java doc comment here, so users of this library would know the min and the max angle?